qemu/tests/virtio_scsi_mq.py

```diff
@@ -216,7 +216,7 @@ def wmi_facility_test(session):
 
     cpu_irq_map = {}
     timeout = float(params.get("login_timeout", 240))
-    num_queues = str(utils_cpu.online_cpus_count())
+    num_queues = str(params['vcpu_maxcpus'])
```

Comment:
no need for str (), params['vcpu_maxcpus']  type is str.

Reply:
updated